experiments/tester/src/RNTester/TestComponents/Callout/CalloutTest.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { ScreenRect, Text, View } from 'react-native';
+import { Button, Callout, Separator } from '@fluentui/react-native';
+import { fabricTesterStyles } from '../Common/styles';
+
+export const CalloutTest: React.FunctionComponent<{}> = () => {
+  const [showStandardCallout, setShowStandardCallout] = React.useState(false);
+  const [showCustomizedCallout, setShowCustomizedCallout] = React.useState(false);
+
+  const stdBtnRef = React.useRef<Text>(null);
+
+  const custBtnRef = React.useRef<Text>(null);
+
+  const toggleShowStandardCallout = React.useCallback(() => {
+    setShowStandardCallout(!showStandardCallout);
+  }, [showStandardCallout, setShowStandardCallout]);
+
+  const toggleShowStandardCalloutInner = React.useCallback(() => {
+    setShowStandardCallout(!showStandardCallout);
+  }, [showStandardCallout, setShowStandardCallout]);
+
+  const toggleShowCustomizedCallout = React.useCallback(() => {
+    setShowCustomizedCallout(!showCustomizedCallout);
+  }, [showCustomizedCallout, setShowCustomizedCallout]);
+
+  const toggleShowCustomizedCalloutInner = React.useCallback(() => {
+    setShowCustomizedCallout(!showCustomizedCallout);
+  }, [showCustomizedCallout, setShowCustomizedCallout]);
+
+  const myRect: ScreenRect = { screenX: 10, screenY: 10, width: 100, height: 100 };
+  return (
+    <View>
+      <Text ref={stdBtnRef} style={fabricTesterStyles.testSection}>
```

Comment:
I'm still not clear what is this ref for. Do you need to access `stdBtnRef` somewhere? It doesn't look like you do.